pages/wg/outreach.md

```diff
@@ -0,0 +1,88 @@
+initiatives for future work.
+
+## Existing Efforts
+
+### RSE Stories
+
+Created by Vanessa Sochat, the [RSE Stories podcast](https://us-rse.org/rse-stories/)
+aims to cast a spotlight on the differing backgrounds and paths professionals
+have taken to become RSEs and help shape our understanding of what an RSE really is.
+
+The podcast has existed since 2019 and is looking for more volunteers both for
+interviewees and production assistance.
+
+### Conference and Workshop Participation
+
+The US-RSE has had a presence at several conferences and workshops throughout
+its lifetime. For example:
+
+- Supercomputing 2021: [RSE-in-HPC Workshop](https://us-rse.org/rse-hpc-2021/), Student Program Orientation
+- PEARC21: [RSE Career Panel](https://us-rse.org/2021-07-20-pearc21-panel/)
+- Supercomputing 2020: [RSE-in-HPC Workshop](https://us-rse.org/rse-hpc-2020/)
+- PEARC20: [Research Software Engineers Community Workshop](https://us-rse.org/events/2020/2020-07-pearc20/)
+- PEARC19: Building a Community of Research Software Engineers Birds of a Feather - [Proposal](https://ssl.linklings.net/conferences/pearc/pearc19_program/views/includes/files/bof111s1.pdf), [Event](https://us-rse.org/events/2019/pearc-2019/)
+- EDUCAUSE 2019: [Supporting Research Software and Research Software Engineers Poster](https://events.educause.edu/annual-conference/2019/agenda/supporting-research-software-development)
+
+### External Collaboration
+
+The US-RSE has collaborated with other organizations in the past for events
+such as the [Early Career Panel](https://academicdatascience.org/cdn/early-career-panel)
+([Academic Data Science Alliance](https://academicdatascience.org/)).
+
+## Future Directions
+
+### Conference and Workshop Participation
+
+US-RSE will continue to engage in conferences and workshops. We aim to
+broaden our presence at previously unrepresented conferences to include
+domain-specific conferences and general technical conferences.
+
+Towards this goal, the Outreach working group is putting together a call for
+volunteers for a [SciPy 2022](https://www.scipy2022.scipy.org/) Birds of a
+Feather RSE panel. If you are interested in participating as a panel member,
+please fill out [this form](https://forms.gle/PUcfcD7LnKVXnZfJ8).
+
+### Customizable RSE Introduction Template
+
+Many members of US-RSE have spoken in diverse environments to a variety of
+audiences about US-RSE. There has been no standardized presentation or
+template for ad-hoc events such as these.
+
+To address this gap, the Outreach working group
+aims to create an "Ad-hoc US-RSE Overview Template" that is customizable for
+different audiences but provides a basic set of standardized content.
+
+### Career Counseling
+
+As Research Software Engineering is still a relatively new professional
+designation, the Outreach working group would like to broaden awareness of
+paths that lead to RSEs throughout colleges and high schools.
```

Comment:
not graduate school? I think grad school maybe comes before these two? And high school? Our frontal lobes don't even finish developing until early 20s lol.

Reply:
In my brain, "colleges" covers grad and undergrad. But if that's not universally true, happy to change the language.

Reply:
I would definitely put college and universities - strangely I've never considered grad school == college. And I think in the UK college _is_ called "University" so we want to hit all the right words!

Reply:
It's complicated in the UK :smile: but these days it's indeed true that places one might study software engineering will tend to now call themselves universities or at least be affiliated with one for that course. Historically it was a much more mixed bag.

Reply:
(Illustrating the diversity is of course places like Singapore Polytechnic.)